### PR TITLE
 Bump in conda version and mamba solver 

### PR DESCRIPTION
**Pull request by dtpryce**
- Bump conda to 22.11.1
- Install and set solver to libmamba
- Tested with a custom runtime build - much much faster
- Tested running the server based on docker image in V2
- Removed the quiet from the build after devs complained they wanted it back there but not from server starting like #1296 

Should solve #1297

### CODE REVIEW

File: mlserver/cli/constants.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
 DockerfileName = "Dockerfile"
 DockerfileTemplate = """
-FROM continuumio/miniconda3:4.12.0 AS env-builder
+FROM continuumio/miniconda3:22.11.1 AS env-builder
```
Comment (adriangonz) [inline mlserver/cli/constants.py:3]: I just noticed that the main `Dockerfile` actually uses `23.1.0`, so should we use that instead to make sure they're aligned?

**Review by adriangonz — COMMENTED** [review]
Nice one @dtpryce! :rocket: 

Added a small comment around the Conda version, but everything else looks good! Once this is in, we may also look into enabling libmamba at the base `Dockerfile` image.

File: mlserver/cli/constants.py (hunk 1 above)
Comment (dtpryce) [reply mlserver/cli/constants.py:3]: I wasn't able to test with that version of the docker image on a linux host machine and got some awful c++ issues locally with our custom runtimes (might be me trying to test on a Mac host but not entirely sure) ... still want me to bump to highest conda image?

Comment (adriangonz) [reply mlserver/cli/constants.py:3]: Fair enough - in that case let's go ahead with the PR as-is :+1:

Comment (dtpryce) [reply mlserver/cli/constants.py:3]: I would add one thing to consider for the future here ... image CVEs are becoming a big part of our work so having both build and inference image at the highest python and even debian as well as library versions is becoming something we focus on. I can see you guys being super proactive on lib CVEs via dependabot but this is a level below that to consider also.

**Pull request merged**